Wikibase/Tests/Namespaces/UnusedUse.php

```diff
@@ -58,6 +58,10 @@ private function process( array $arg, AnotherBadCapiTaliZation $arg ) {
 		/* @var $var bool MentionedInComment should not count as usage */
 		$withAComment = new WithAComment();
 		$withADocComment = new WithADocComment();
+
+		// These are strings for the tokenizer, but not class names
```

Comment:
I was a bit confused here at first… WDYT about changing “These” to “These 'unused' tokens”? At first I thought it was referring to `PrecededByDoubleColon` and `$precededByObjectOperator`.

Reply:
All fixed. Thanks for the suggestions!